subscription.go

```diff
@@ -193,7 +192,15 @@ func (r *marathonClient) registerSSESubscription() error {
 	return nil
 }
 
-// Unsubscribe removes ourselves from Marathon's callback facility
+// Subscribe adds a URL to Marathon's callback facility
+//	url		: the url you wish to subscribe
+func (r *marathonClient) Subscribe(callbackURL string) error {
+	uri := fmt.Sprintf("%s?callbackUrl=%s", marathonAPISubscription, callbackURL)
+	return r.apiPost(uri, "", nil)
+
+}
+
+// Unsubscribe removes a URL from Marathon's callback facility
 //	url		: the url you wish to unsubscribe
```

Comment:
Name of the variable is actually `callbackURL`.

Should also say `URL` in the description.